Userland/Libraries/LibGfx/EdgeFlagPathRasterizer.h

```diff
@@ -50,6 +50,21 @@ struct AA {
     }
 };
 
+// 2-bit sample (mainly for use with NoAA).
```

Comment:
(here too)

Reply:
Not sure how it applies here? :sweat_smile: